testkit/metrics/jvm/src/main/scala/org/typelevel/otel4s/testkit/metrics/MetricsSdk.scala

```diff
@@ -18,6 +18,8 @@ package org.typelevel.otel4s
 package testkit
 package metrics
 
+import _root_.java.{lang => jl}
+import _root_.java.{util => ju}
```

Comment:
Unidoc aggregates all the sources before running scaladoc, so these imports were conflicting with the `org.typelevel.otel4s.java` package.